modules/swagger-codegen/src/main/resources/typescript-angular/configuration.mustache

```diff
@@ -23,4 +23,53 @@ export class Configuration {
         this.basePath = configurationParameters.basePath;
         this.withCredentials = configurationParameters.withCredentials;
     }
+
+    /**
+     * Select the correct content-type to use for a request.
+     * Uses {@link Configuration#isJsonMime} to determine the correct content-type.
+     * If no content type is found return the first found type if the contentTypes is not empty
+     * @param {string[]} contentTypes - the array of content types that are available for selection
+     * @returns {string} the selected content-type or <code>undefined</code> if no selection could be made.
+     */
+    public selectHeaderContentType (contentTypes: string[]): string {
+        if (contentTypes.length == 0)
+            return undefined;
+
+        let type = contentTypes.find(x => this.isJsonMime(x));
+        if (type === undefined) 
```

Comment:
I suggest using block braces `{` and `}` as usually linting rules discourage if statements without braces